spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -294,12 +295,7 @@ def main(args=None):
     if arguments.vertfile is not None:
         fname_vert_levels = arguments.vertfile
         if arguments.vert is not None:
-            # Get dimensions of input and the vertebral labeling file
-            nx, ny, nz = Image(fname_segmentation).change_orientation('RPI').data.shape
-            nx_vertebral, ny_vertebral, nz_vertebral =  Image(fname_vert_levels).change_orientation('RPI').data.shape
-
-            # Check dimensions consistency between the input and the vertebral labeling file
-            if (nx, ny, nz) != (nx_vertebral, ny_vertebral, nz_vertebral):
+            if check_dimensions_match(input_im=Image(fname_segmentation).change_orientation('RPI'), im_vertebral_labeling=Image(fname_vert_levels).change_orientation('RPI')):
                 print('\nERROR: Input data and vertebral labeling file DO NOT HAVE SAME DIMENSIONS.', 1, type='error')
```

Comment:
ditto

Reply:
I do the `check_dimensions_match` in `aggregate_per_slice_or_level()`.
Please see the comment: https://github.com/neuropoly/spinalcordtoolbox/pull/3120#issuecomment-751926217